src/components/Form.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+
+class Form extends Component {
+  state = { name: String(), description: String(), inCart: false };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    const { addNewItem } = this.props;
+    this.checkForDuplicates(this.state.name)
+      ? alert('Item already exists. Please enter a different item.')
+      : addNewItem && addNewItem(this.state);
+    this.setState({ name: String(), description: String() });
+  };
+
+  handleInputChange = field => e => this.setState({ [field]: e.target.value });
+
+  checkForDuplicates = newItem => this.props.list.find(item => item['name'].toLowerCase() === newItem.toLowerCase());
```

Comment:
`item.name`. Square bracket notation is usually meant for interpolating variables `item[foo]` or handling logic like `myArray[itemExists ? 'push' : 'pop'](item)`